src/test/java/app/studnicki/ox/RoundTest.java

```diff
@@ -0,0 +1,80 @@
+package app.studnicki.ox;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class RoundTest {
```

Comment:
dataProvider maybe?

Reply:
In the next tests I think it should be very helpful.